package/MDAnalysis/analysis/hbonds/hbond_analysis.py

```diff
@@ -79,6 +83,9 @@
    MDAnalysis simply subtract 1. For instance, to find an atom in
    :attr:`Universe.atoms` by *index* from the output one would use
    ``u.atoms[index-1]``.
+   .. versionchanged:: 0.15.0
```

Comment:
Use the sphinx [deprecated](http://www.sphinx-doc.org/en/stable/markup/para.html#directive-deprecated) markup:
```rst
.. deprecated:: 0.16.0
   This feature is being deprecated in favor of zero-based indices 
   and is targeted  for removal in 0.16.0. Use `donor_index` and
   `acceptors_index` instead.
```